org-gcal.el

```diff
@@ -2282,6 +2160,24 @@ non-nil."
     (plist-get plst :year))))
 
 
+(defun org-gcal-reload-client-id-secret ()
+  "Setup OAuth2 authentication after setting client ID and secret."
+  (interactive)
+  (add-to-list
+   'oauth2-auto-additional-providers-alist
+   `(org-gcal
+     (authorize_url . "https://accounts.google.com/o/oauth2/auth")
+     (token_url . "https://oauth2.googleapis.com/token")
+     (scope . "https://www.googleapis.com/auth/calendar")
```

Comment:
Note also that the [PR 9](https://github.com/rhaps0dy/emacs-oauth2-auto/pull/9), which I'm going to merge at some point, changes the format to `(available_scopes . (calendar . "https://www.googleapis.com/auth/calendar"))`